auth/openid_connect.adoc

```diff
@@ -0,0 +1,213 @@
+
+The following _Group Membership_ mapper must be manually created:
+
+[options="header",cols="8*^"]
+|============================================================================================================
+| Name                 | Consent Required | Mapper Type      | Token Claim Name | Full group path | Add to ID token | Add to access token | Add to userinfo
+| groups               |   OFF            | Group Membership | groups           | OFF             | ON              | OFF                 | OFF
+|============================================================================================================
+
+The following _User Session Note_ mappers must be manually created:
+
+[options="header",cols="8*^"]
+|============================================================================================================
+| Name                 | Consent Required | Mapper Type       | User Session Note | Token Claim Name | Claim JSON Type | Add to ID token | Add to access token
+| Client Host          |   OFF            | User Session Note | clientHost        | clientHost       | String          | ON              | ON
+| Client ID            |   OFF            | User Session Note | clientId          | clientId         | String          | ON              | ON
+| Client IP Address    |   OFF            | User Session Note | clientAddress     | clientAddress    | String          | ON              | ON
+|============================================================================================================
+
+[[configuring-the-administrative-ui]]
+== Configuring the Administrative UI
+
+After having configured Apache for OIDC, the next step is to update the Appliance Administrative UI
+to be OIDC aware and function accordingly.
+
+Login as admin, then in _Configure->Configuration->Authentication_
+
+* Set mode to External (httpd)
+* Check: _Provider Type: Enable Enable OpenID-Connect_ - This enables the OIDC login button on the login screen, that redirects
+to the OIDC protected page for authentication, and supports the OIDC logout process.
+* Optional: Check: _Enable Single Signon_ - With this option enabled, initial access to the Appliance Administrative UI will
+redirect to the OIDC Identity Provider authentication screen.  Note that logouts from the Appliance will
+return the user to the Appliance login screen allowing them to login as admin unless _Disable Local Login_
+is checked below.
+* Optional: Check: _Disable Local Login_ - Do this *only* if you need to disable _admin_ login to appliance and only
+allow OIDC based authentication.  Note that if there are issues with the Identity Provider or you need
+admin access to the appliance you won't be able to login until you re-enable the Local Login
+as described below.
+* Check: _Get User Groups from External Authentication (httpd)_
+* Click Save.
+
+The above steps need to be done on each UI enabled appliance.
+
+in _Configure->Configuration->Access Control_
```

Comment:
The formatting could make this step a little bit unclear, so I suggest editing this line to something like this, so it's definitely pointing to the instruction after it:

In _Configure->Configuration->Access Control_:

Reply:
Thank you @dayleparker  Will do.